src/components/display.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function display({ value }) {
+  return (
+    <div className='display'>
+      <div className='display-value'>
+        {value}
+      </div>
+    </div>
+  );
+}
+
+export default display;
```

Comment:
- The result prop should be a String.
- The default value of the result prop should be 0.

Try to do some research regarding `proptypes` and `defaultProps `
and solve these two issues